### ISSUE TITLE
System.Xaml is missing

### ISSUE THREAD

**Issue by ewinnington** [Microsoft/referencesource#39]
The source for System.Xaml is missing from reference source. 

As discussed in the [Port System.XAML](https://github.com/dotnet/corefx/issues/5766#issuecomment-320724209) issue, having the .net framework reference source would allow the community to move forward with supporting xaml based scenarios, specifically the parsing of Workflow Foundation Xaml serialized documents.

**Comment by jakesays** [comment]
I tried and tried to get the source to system.xaml. no luck. Apparently it contains corporate trade secrets or something.

**Comment by poizan42** [comment]
It has just been made available over at https://github.com/dotnet/wpf

**Comment by ewinnington** [comment]
Wonderful! Should I close this issue?

**Comment by terrajobst** [comment]
> Wonderful! Should I close this issue?

No, because this is orthogonal. We should still add the files here.
